lxd/api_cluster.go

```diff
@@ -1255,6 +1255,16 @@ func clusterNodeDelete(d *Daemon, r *http.Request) response.Response {
 		force = 0
 	}
 
+	err = autoSyncImages(d.ctx, d)
```

Comment:
I think this can be moved down to after:
```
	logger.Debugf("Deleting member %s from cluster (force=%d)", name, force)
```

And we should add another run of `autoSyncImages` just before the end of the function to replicate any image that was lost with that node back to full replication.